SampleApp/src/main/java/ly/kite/sample/MainActivity.java

```diff
@@ -151,6 +151,8 @@ protected void onCreate( Bundle savedInstanceState )
     {
     super.onCreate( savedInstanceState );
 
+    KiteSDK.ENCRYPTION_KEY = REPLACE_DETAILS_HERE;
```

Comment:
Unfortunately we can't expect the user to set the ENCRYPTION_KEY like this. I think the best thing to do would be to set it by default to the users public API Key when they set that within the SDK.